src/Azs.Compute.Admin/docs/Update-AzsComputeGlobalFeatureSetting.md

```diff
@@ -0,0 +1,252 @@
+---
+external help file:
+Module Name: Azs.Compute.Admin
+online version: https://docs.microsoft.com/en-us/powershell/module/azs.compute.admin/update-azscomputeglobalfeaturesetting
+schema: 2.0.0
+---
+
+# Update-AzsComputeGlobalFeatureSetting
+
+## SYNOPSIS
+Update the feature settings.
+
+## SYNTAX
+
+### UpdateExpanded (Default)
+```
+Update-AzsComputeGlobalFeatureSetting -FeatureName <String> [-Location <String>] [-SubscriptionId <String>]
+ [-GlobalFeatureState <GlobalFeatureState>] [-DefaultProfile <PSObject>] [-PassThru] [-Confirm] [-WhatIf]
+ [<CommonParameters>]
+```
+
+### Update
+```
+Update-AzsComputeGlobalFeatureSetting -FeatureName <String> -GlobalFeatureSetting <IGlobalFeatureSettings>
+ [-Location <String>] [-SubscriptionId <String>] [-DefaultProfile <PSObject>] [-PassThru] [-Confirm] [-WhatIf]
+ [<CommonParameters>]
+```
+
+### UpdateViaIdentity
+```
+Update-AzsComputeGlobalFeatureSetting -InputObject <IComputeAdminIdentity>
+ -GlobalFeatureSetting <IGlobalFeatureSettings> [-DefaultProfile <PSObject>] [-PassThru] [-Confirm] [-WhatIf]
+ [<CommonParameters>]
+```
+
+### UpdateViaIdentityExpanded
+```
+Update-AzsComputeGlobalFeatureSetting -InputObject <IComputeAdminIdentity>
+ [-GlobalFeatureState <GlobalFeatureState>] [-DefaultProfile <PSObject>] [-PassThru] [-Confirm] [-WhatIf]
+ [<CommonParameters>]
+```
+
+## DESCRIPTION
+Update the feature settings.
+
+## EXAMPLES
+
+### -------------------------- EXAMPLE 1 --------------------------
+```powershell
+Update-AzsComputeGlobalFeatureSetting -FeatureName Microsoft.Compute.EmergencyVMAccess -GlobalFeatureState Enabled -Location local
+```
+
+
+
+## PARAMETERS
+
+### -DefaultProfile
+The credentials, account, tenant, and subscription used for communication with Azure.
+
+```yaml
+Type: System.Management.Automation.PSObject
+Parameter Sets: (All)
+Aliases: AzureRMContext, AzureCredential
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -FeatureName
+Name of the feature.
+
+```yaml
+Type: System.String
+Parameter Sets: Update, UpdateExpanded
+Aliases:
+
+Required: True
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -GlobalFeatureSetting
```

Comment:
add documentation on meaning of 3 enum values: Enabled, Disabled, and TenantSubscriptionLevel. Enabled/Disabled will override features that are enabled with tenant subscription ID. TenantSubscriptionLevel will defer feature enablement to per tenant subscription ID enablement.